alacritty/Cargo.toml

```diff
@@ -33,6 +33,9 @@ rustc_tools_util = "0.2.0"
 [target.'cfg(not(windows))'.dependencies]
 xdg = "2"
 
+[target.'cfg(target_os = "macos")'.dependencies]
+cocoa = { git = "https://github.com/servo/core-foundation-rs", branch = "master" }
```

Comment:
This obvious needs to change, once they tag and release the new version which has the patch i've commited.